cea/interfaces/dashboard/api/project.py

```diff
@@ -22,92 +22,89 @@
-        config = current_app.cea_config
-        payload = api.payload
-
-        if 'path' in payload:
-            path = payload['path']
-            if os.path.exists(path):
-                config.project = path
-                if 'scenario' not in payload:
-                    config.scenario_name = ''
-                    config.save()
-                    return {'message': 'Project path changed'}
-            else:
-                abort(400, 'Path given does not exist')
+        project_path = request.args.get('path')
+        if project_path is None:
+            config = current_app.cea_config
+            scenario = config.scenario_name
+        else:
+            if not os.path.exists(project_path):
+                abort(400, 'Project path: "{project_path}" does not exist'.format(project_path=project_path))
+            # Prevent changing current_app config
+            config = cea.config.Configuration()
+            config.project = project_path
+            scenario = None
 
-        if 'scenario' in payload:
-            scenario = payload['scenario']
-            choices = list_scenario_names_for_project(config)
-            if scenario in choices:
-                config.scenario_name = scenario
-                config.save()
-                return {'message': 'Scenario changed'}
-            else:
-                abort(400, 'Scenario does not exist', choices=choices)
+        return {'name': os.path.basename(config.project), 'path': config.project, 'scenario': scenario,
+                'scenarios': list_scenario_names_for_project(config)}
 
-    @api.doc(body=NEW_PROJECT_MODEL, responses={200: 'Success'})
     def post(self):
-        """Create new project"""
-        config = current_app.cea_config
-        payload = api.payload
+        """Create new project folder"""
+        name = api.payload.get('name')
+        project_path = api.payload.get('path')
 
-        if 'path' in payload:
-            path = payload['path']
-            name = payload['name']
-            project_path = os.path.join(path, name)
+        if name is not None and project_path is not None:
+            project_path = os.path.join(project_path, name)
             try:
                 os.makedirs(project_path)
             except OSError as e:
                 abort(400, str(e))
 
-            config.project = project_path
-            config.scenario_name = ''
-            config.save()
+            return {'message': 'Project folder created', 'path': project_path}
+        else:
+            abort(400, 'Parameters not valid - Project Name: {name}, Project Path: {project_path}'.format(
+                name=name, project_path=project_path
+            ))
 
-            return {'message': 'Project created at {}'.format(project_path)}
+    @api.expect(SCENARIO_PATH_MODEL)
+    def put(self):
+        """Update Project info in config"""
+        config = current_app.cea_config
+        path = api.payload.get('path')
```

Comment:
why aren't we calling this `project-path` or `project`? that way, we use the same names as in the config files...